src/components/Functions/axisFormat.ts

```diff
@@ -11,10 +11,25 @@ export const formatDateAsDayMonth = (i: string): string => {
     return `${parts[1]}/${parts[2]}`;
 }
 
-export const formatNumberAsK = (n: string | number = 0): string =>
-    (Math.abs(+n) > 1000)
-        ? `${(+n / 1000).toFixed(1)}k`
-        : `${(+n).toFixed(0)}`;
+export const formatNumberAsK = (n: string | number = 0): string => {
+    if (+n === 0) {
```

Comment:
The preferred method to convert something to string is  `{stringVar}`. Can we use it here too (and bellow).

I prefer having a single return statement without `{` and `}` inlined.